app/presenters/crime_application_presenter.rb

```diff
@@ -14,6 +14,8 @@ def applicant_dob
   end
 
   def applicant_name
+    return unless applicant?
```

Comment:
Is this a boy scout style addition unrelated to the general refactor you're doing here in relation to removing the concept of Decorators?